Feature/Remote Control - Allow Multiple Modules per Module Type (#171)
smartdevicelink/sdl_hmi_integration_guidelines: docs/RC/SetGlobalProperties/index.md

## SetGlobalProperties

Type
: Function

Sender
: SDL

Purpose
: Inform HMI of user location properties set by the application for RC modules allocation purposes

### Description

> SDL requests to set the values of user location sent by RC application. The default `userLocation` should be `DRIVER`.  

Reviewer: ```suggestion
SDL requests to set the value of user location sent by a RC application. The default `userLocation` should be `DRIVER`.  
```

> In case driver's seat is out of the service area of a module, driver's application should be able to control any free module that is allowed by policies.

Reviewer: ```suggestion
The driver's application should be able to control any free module that is allowed by policies(even if the driver's seat is outside the service area of a module).
```

### Request
#### Behavior

!!! MUST   

1. Store the userLocation and associate it with appID.

> 2. Respond to the request.

Reviewer: Probably not necessary to state that the HMI must respond to the RPC request.

Reviewer 2: removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/171/commits/de249c04759674b8befb3be7d380a6d5e27820b9
